linera-core/src/unit_tests/client_tests.rs

```diff
@@ -1933,6 +2051,12 @@ async fn test_memory_propose_pending_block() -> Result<(), anyhow::Error> {
     run_test_propose_pending_block(MakeMemoryStorage::default()).await
 }
 
+#[cfg(not(target_arch = "wasm32"))]
+#[test(tokio::test)]
+async fn test_service_propose_pending_block() -> Result<(), anyhow::Error> {
+    run_test_propose_pending_block(MakeServiceStorage::new("127.0.0.1:9019")).await
```

Comment:
There's also `port-selector`, which might remove the need to manually maintain a list of distinct ports for the tests, but I'm not sure how reliable it is.

Reply:
Ok, I implemented that.